README.md

```diff
@@ -1,30 +1,41 @@
-# Hyperledger Ursa
+<p align="center"><h1>Hyperledger URSA</h1></p><br>
+---
+
+- [Introduction](#introduction)
+- [Features](#features)
+    - [Libursa](#Libursa)
+    - [Libzmix](#libzmix)
+- [Dependencies](#dependencies)
+- [Building from source](#building-from-source)
+- [Contributing](#contributing)
+
+# Introduction
 
 Ursa was created because people in the Hyperledger community realized that it would save time and effort and improve security if we all collaborated on our cryptographic code. Since cryptographic APIs are relatively straightforward to define, it would be possible for many different projects to utilize the same code without too much difficulty.
 
 First and foremost, we hope in the long run that Ursa provides open-source blockchain developers with reliable, secure, easy-to-use, and pluggable cryptographic implementations.
 
+# Features
 The major artifacts of Ursa are:
 - C-callable library interface
 - Rust crate
 
-All bugs, stories, and backlog for this project are managed through Hyperledger's Jira in project IS (note that regular Ursa tickets are in the URSA project). Also, join us on [Hyperledger Rocket.Chat](https://chat.hyperledger.org) at #ursa to discuss. The ursa group also meets biweekly on Wednesday's at 7 AM PST at https://zoom.us/my/hyperledger.community. The meeting notes are available [here](https://docs.google.com/document/d/1Z_8o8k_PFRM4XfZyv9jH1_-IyN0CsCMI2JlrGsCX378/edit).
-
-Major modifications to ursa are submitted as RFCs to the [Ursa RFC repo](https://github.com/hyperledger/ursa-rfcs). 
-
 Ursa is divided into two sub libraries: libursa and libzmix.
 
-Libursa is designed for cryptographic primitives like simple digital signatures, encryption schemes, and key exchange.
+## Libursa
+
+Designed for cryptographic primitives like simple digital signatures, encryption schemes, and key exchange.
 
-Libzmix offers a generic way to create zero-knowledge proofs, proving statements about multiple cryptographic building blocks, containing signatures, commitments, and verifiable encryption. Libzmix uses many of the building blocks found in Libursa.
+## Libzmix
+A generic way to create zero-knowledge proofs, proving statements about multiple cryptographic building blocks, containing signatures, commitments, and verifiable encryption. Libzmix uses many of the building blocks found in Libursa.
 
-## Dependencies
+# Dependencies
 
 Ursa and zmix use the following external dependencies:
 
-- libsodium 1.0.14 (Written in C)
-- openssl 1.1.0j or newer (Written in C)
-- libsecp256k1 (Written in C)
+- [libsodium 1.0.14](https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz) (Written in C)
+- [openssl 1.1.0j](https://www.openssl.org/source/openssl-1.1.0j.tar.gz) or newer (Written in C)
+- [libsecp256k1](https://github.com/bitcoin-core/secp256k1) (Written in C)
```

Comment:
Shouldn't the `num-bigint` dependency be mentioned too?

Reply:
@mikelodder7 ?

Reply:
No